### PR TITLE
Adding BCeID lookup endpoint, get name data from BCeID

### PR DESCRIPTION
**Pull request by alexjoybc**
# Description

This PR includes the following proposed change(s):

- Adding BCeID lookup endpoint, get name data from BCeID
- Get name from a bceid lookup endpoint instead of from original token

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation

## How Has This Been Tested?

- local
- jest
- storybook

### CODE REVIEW

File: src/backend/efiling-api/jag-efiling-api.yaml [hunk 1]
```diff
@@ -231,6 +231,24 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/UserFullDetails"
+  "/bceidAccount":
+    get:
+      summary: Get a Bceid account
```
Comment (sdevalapurkar-bcgov) [inline src/backend/efiling-api/jag-efiling-api.yaml:236]: nit:
```suggestion
      summary: Get a BCeID account
```

File: src/backend/efiling-api/jag-efiling-api.yaml [hunk 2]
```diff
@@ -231,6 +231,24 @@ paths:
...
+      operationId: GetBceidAccount
+      description: Retrive bceid user account information
```
Comment (sdevalapurkar-bcgov) [inline src/backend/efiling-api/jag-efiling-api.yaml:238]: spelling:
```suggestion
      description: Retrieve BCeID user account information
```

File: src/backend/efiling-api/jag-efiling-api.yaml [hunk 3]
```diff
@@ -292,7 +310,17 @@ components:
           description: the account type
         identifier:
           type: string
-          description: the account identifier
+          description: the account identifie
```
Comment (sdevalapurkar-bcgov) [inline src/backend/efiling-api/jag-efiling-api.yaml:313]: spelling:
```suggestion
          description: the account identifier
```

File: src/backend/efiling-api/jag-efiling-api.yaml [hunk 4]
```diff
@@ -292,7 +310,17 @@ components:
...
+    BceidAccount:
+      type: object
+      description: bceid account details
```
Comment (sdevalapurkar-bcgov) [inline src/backend/efiling-api/jag-efiling-api.yaml:316]:
```suggestion
      description: BCeID account details
```

File: src/backend/efiling-api/jag-efiling-api.yaml (hunk 2 above)
Comment (akroon3r) [reply src/backend/efiling-api/jag-efiling-api.yaml]: 🧺

**Comment by sdevalapurkar-bcgov** [comment]
@adodge-bcg will leave it to you to do final approve/merge

File: src/frontend/efiling-frontend/src/components/page/home/Home.js [hunk 5]
```diff
@@ -81,13 +75,25 @@ const checkCSOAccountStatus = (
         ).identifier;
         sessionStorage.setItem("csoAccountId", csoAccountIdentifier);
         setCsoAccountStatus({ isNew: false, exists: true });
+      } else {
+        axios
+          .get("/bceidAccount")
```
Comment (adodge-bcg) [inline src/frontend/efiling-frontend/src/components/page/home/Home.js:80]: Do we not need to pass a submission id or anything to this endpoint?

Comment (sdevalapurkar-bcgov) [reply src/frontend/efiling-frontend/src/components/page/home/Home.js:80]: nope!

**Comment by sdevalapurkar-bcgov** [comment]
4 approvals!! @TayGov @akroon3r

**Pull request merged**